src/stability_sdk/animation.py

```diff
@@ -118,9 +108,8 @@ class ColorSettings(param.Parameterized):
     lightness_curve = param.String(default="0:(0.0)")
 
 
-# TO DO: change to a generic `depth_weight` rather than specifying model name in the parameter
-class DepthwarpSettings(param.Parameterized):
-    midas_weight = param.Number(default=0.3, softbounds=(0,1), doc="Strength of depth model influence.")
+class DepthSettings(param.Parameterized):
+    depth_model_weight = param.Number(default=0.3, softbounds=(0,1), doc="Blend factor between Adabins and MiDaS depth models.")
```

Comment:
I think it's best to use `AdaBins` capitalization (in part to match the careful capitalization style used for MiDaS here)

Reply:
oops, yes you're correct - will fix